index.html

```diff
@@ -1,65 +1,58 @@
 <!DOCTYPE html>
 <html>
-    <head>
-        <meta charset="utf-8">
-        <meta http-equiv="X-UA-Compatible" content="IE=edge">
-        <meta name="viewport" content="width=device-width, initial-scale=1">
 
-        <title>WNYC_TEST</title>
-        <link rel="apple-touch-icon" sizes="120x120" href="img/apple-touch-icon.png">
-        <link rel="icon" type="img/png" href="img/favicon-32x32.png" sizes="32x32">
-        <link rel="icon" type="img/png" href="img/favicon-16x16.png" sizes="16x16">
-        <!-- <link rel="manifest" href="img/manifest.json"> -->
-        <link rel="mask-icon" href="img/safari-pinned-tab.svg" color="#5bbad5">
-        <meta name="theme-color" content="#ffffff">
-
-        <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.6/css/bootstrap.min.css">
-        <link rel="stylesheet" href="style.css" media="screen" charset="utf-8">
-        <link rel="stylesheet" href="icons.css" media="screen" charset="utf-8">
-    </head>
-    <body>
-        <main>
-            <div class="col-md-12 text-center">
-                <section class="top-section">
-                    <div id="topStories">Democrats Fail to Make Gains<br> in NY State and Federal Races</div>
-                    <div id="source">from WNYC news</div>
-                    <div class="wrapper">
-                         <button class="button">
-                            Listen Live
-                         </button>
-                    </div>
-
-                </section>
+<head>
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <title>WNYC_TEST</title>
+    <link rel="apple-touch-icon" sizes="120x120" href="img/apple-touch-icon.png">
+    <link rel="mask-icon" href="img/safari-pinned-tab.svg" color="#5bbad5">
+    <meta name="theme-color" content="#ffffff">
+    <link rel="stylesheet" href="//cdnjs.cloudflare.com/ajax/libs/flexboxgrid/6.3.1/flexboxgrid.min.css" type="text/css">
+    <link rel="stylesheet" href="style.css" media="screen" charset="utf-8">
+</head>
+
+<body>
+    <div class="main">
+        <section class="main-section">
+            <div id="topStories1"><a class="top-title" href="http://www.wnyc.org/story/rebels-agree-leave-last-strongholds-aleppo-amidst-civilian-deaths-outrage/">Rebels to Leave Last Strongholds <br>in Aleppo Amidst Civilian Deaths Outrage</a>
             </div>
-
-            <div class="row">
-                <div class="col-md-6">
-                    <section id="logo-section" class="bottom-section"> <!--   previously twitter-section -->
-
-
-                     <!--    <h4>logo here</h> -->
-                     <img src="/../img/wnyclogo.png" alt="Logo" style="width:124px;height:30px;">
-                    </section>
+            <div id="middle-container1">
+                <div id="left-arrow">
+                    <a class="previous" href="index2.html"></a>
```

Comment:
Be more explicit with the page names. Maybe the second page name could be `top-stories.html` for instance.